app/models/transaction_category.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: transaction_categories
+#
+#  id         :bigint           not null, primary key
+#  slug       :citext           not null
```

Comment:
What are we planning on using slugs for?

Reply:
I see they're similar to labels - what's the thought behind the name difference?

Reply:
That feels good to me - thanks for the extra context